mixins/framework.js

```diff
@@ -28,25 +34,59 @@ module.exports = {
 						'**/samples/**/*.*',
 						'**/tests/**/*.*'
```

Comment:
Is `*.*` needed or is `*` enough?  Do we need to ignore `coverage` here, too?

Reply:
This is within the app's node_modules, so I'd hope we weren't publishing coverage files, heh.